src/idsk/components/header-web/template.njk

```diff
@@ -0,0 +1,315 @@
+                                                        {% for language in brand.languages | sort(attribute='text') %}
+                                                        <li class="idsk-header-web__brand-language-list-item">
+                                                                <a class="govuk-link idsk-header-web__brand-language-list-item-link" title="{{ language.text }}" href="{{ language.link }}">
+                                                                        {{ language.text }}
+                                                                </a>
+                                                        </li>
+                                                        {% endfor %}
+                                                </ul>  
+                                        </div>
+                                </div>        
+
+                        </div>
+                        <div class="idsk-header-web__brand-dropdown">
+                                <div class="govuk-grid-row">
+                                        <div class="govuk-grid-column-one-half">
+                                                <p class="govuk-body-s">Doména Gov.sk je oficálna</p>
+                                                <p class="govuk-body-s">Slovak government websites always use a .gov.sk domain. Before sharing sensitive information online, make sure.</p>
+                                        </div>
+                                        <div class="govuk-grid-column-one-half">
+                                                <p class="govuk-body-s">Táto stránka je zabezpečená</p>
+                                                <p class="govuk-body-s">Slovak government websites always use a .gov.sk domain. Before sharing sensitive information online, make sure.</p>
+                                        </div>
+                                </div>  
+                        </div>
+                        
+                </div>
+                
+        </div>
+{% endset %}
+
+
+{# MAIN SECTION #}
+{% set main = params.main %}
+{% set mainButtons %}
+        <div class="idsk-header-web__main--buttons">
+        {% if main.loginButton %} 
+                <div class="idsk-header-web__main--login">
+                        {{ idskButton({
+                                "text": main.loginButton.text if main.loginButton.text else 'Prihlásiť sa',
+                                "type": "button",
+                                "classes": main.loginButton.classes if main.loginButton.classes
+                        }) }}
+                        <div class="idsk-header-web__main--login-action">
+                                <img class="idsk-header-web__main--login-action-profile-img" src="{{ main.loginButton.loggedIn.profileImg if main.loginButton.loggedIn.profileImg else '/public/assets/images/header-web/profile.svg' }}" alt="Profile image">
```

Comment:
- '/public/assets/images' nie som si uplne isty ci toto je univerzalny pristup k obrazku.
- plus pri takom to dlhom riadku, neboj sa odsadit attributy na novy riadok, bude to tak prehladnejsie citatelnejsie ako scrollovat riadoky do stran